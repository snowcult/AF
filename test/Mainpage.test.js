import React from 'react';
import { shallow } from 'enzyme';
import MainPage from '../src/components/pages/MainPage';

describe('MainPage component', () => {
  let wrapper;
  const mockProps = {
    searchField: '',
    onSearchChange: jest.fn(),
    robots: [],
    isPending: false,
    onRequestRobots: jest.fn(),
  };

  wrapper = shallow(<MainPage {...mockProps} />);

  it('renders Main page without errors', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
