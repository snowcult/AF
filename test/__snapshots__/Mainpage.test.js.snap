// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPage component renders Main page without errors 1`] = `
<div
  className="tc"
>
  <Header />
  <h2
    className="red"
  >
    Loading...
  </h2>
  ;
</div>
`;
